src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { FILTER_OPTIONS } from '../../utils/constants';
+
+type Props = {
+  todos: Todo[],
+  todosToRender: Todo[]
+  setTodosToRender: (arr: Todo[]) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  todosToRender,
+  setTodosToRender,
+}) => {
+  const [selectedButton, setSelectedButton] = useState('All');
+  const itemsLeft
+  = todos.reduce((acc, item) => (!item.completed ? acc + 1 : acc), 0);
+
+  const filterTodos = (e: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
Consider using the event handler type React.MouseEventHandler<HTMLAnchorElement> to avoid using 'e.currentTarget.innerText'.